data/112/578/988/9/1125789889.geojson

```diff
@@ -211,7 +214,10 @@
     },
     "wof:country":"AT",
     "wof:created":1497290827,
-    "wof:geomhash":"fbe79717e2495b002ac515781addbfbd",
+    "wof:geom_alt":[
```

Comment:
Seems suspicious / duplicative with `src:geom_alt`...

Reply:
Some about the property: https://github.com/whosonfirst/whosonfirst-properties/blob/main/properties/wof/geom_alt.json

Issue: https://github.com/whosonfirst-data/whosonfirst-data/issues/1793

This property is basically the `src:geom_alt` prop, but only displays source names of each alt file.